.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,33 @@
+name: "CodeQL"
+
+on: pull_request
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: ["javascript"]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      # Initializes the CodeQL tools for scanning.
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v1
+        with:
+          languages: ${{ matrix.language }}
```

Comment:
do we need it, if we are using only one specific language?

Reply:
Nope